### ISSUE TITLE
Tests pass when error is thrown

### ISSUE THREAD

**Issue by tomekwi** [scottcorgan/tap-dot#4]
When the tested code throws an error, you get output similar to:

```sh
>  node test | tap-dot

module.js:340
    throw err;
          ^
Error: Cannot find module 'trimmer'
    at <stack trace here>

  0 tests
  0 passed

  Pass!
```

I guess the test should fail, as it does without any reporter, or with [substack/faucet](https://github.com/substack/faucet).

**Comment by scottcorgan** [comment]
Will be fixed in `1.0` release in the next or so. Thanks for the find.

**Comment by tomekwi** [comment]
For the record – for now I'm using [tape-catch](https://www.npmjs.com/package/tape-catch), which does the job as well. But I admit it does feel a bit dirty :)

**Comment by scottcorgan** [comment]
Didn't Tape just release a fix to catch errors?

**Comment by tomekwi** [comment]
Well, it looks as if they used to but they've decided against it: https://github.com/substack/tape#uncaught-exceptions

**Comment by scottcorgan** [comment]
Published as `1.0.0`

**Comment by tomekwi** [comment]
Thank you very much!
